util/are-ids-valid.ts

```diff
@@ -0,0 +1,41 @@
+import { PREFIXES } from '../config/custom-dispatching/delta-context-config';
+import { findFirstSparqlResult } from './sparql-result';
+import { query, sparqlEscapeString } from 'mu';
+
+export async function areIdsValid(
+  rdfType: RDF_TYPE,
+  ids?: Array<string>,
+): Promise<boolean> {
+  if (!ids || ids.length === 0) {
+    return false;
+  }
+
+  const values = ids.map((id) => sparqlEscapeString(id));
+  const countOfExisting = `
+    ${PREFIXES}
```

Comment:
Can I use this? It is from another context im aware but it has what we need :D

Reply:
VERY DANGEROUS. don't ever reuse prefixes from other contexts they don't necessarily mean the same thing

Reply:
Set back the delta-context config and added the prefixes to the areidsvalid method